metadata.yaml

```diff
@@ -43,6 +43,13 @@ provides:
 requires:
   alertmanager:
     interface: alertmanager_dispatch
+  ingress-per-unit:
+    interface: ingress_per_unit
+    limit: 1
+
+peers:
+  loki:
+    interface: loki
```

Comment:
Consider using `peers`, `replicas`, etc.
In juju status,
```
Relation provider
loki:loki
```
seems a bit confusing to me.